ddtrace/contrib/logging/patch.py

```diff
@@ -83,11 +84,9 @@ def _w_makeRecord(func, instance, args, kwargs):
         span = _get_current_span(tracer=config.logging.tracer)
 
     if span:
-        trace_id = span.trace_id
-        if config._128_bit_trace_id_enabled and not config._128_bit_trace_id_logging_enabled:
-            trace_id = span._trace_id_64bits
+        trace_id, span_id = _get_trace_details_for_log_injection(span)
```

Comment:
Question: Why cant we use `config.logging.get_log_correlation_context` in the logging integrations?

Reply:
We can centralize all the log formatting logic in `tracer.get_log_correlation_context` and get rid of `_get_trace_details_for_log_injection`

Reply:
yup, done